olp-cpp-sdk-core/include/olp/core/client/OlpClientSettings.h

```diff
@@ -178,11 +189,25 @@ struct RetrySettings {
 
   /**
    * @brief The backdown policy that should be used for the retry attempts.
+   * 
+   * @deprecated Use \ref backdown_strategy instead.
    *
    * @return The backdown policy.
    */
   BackdownPolicy backdown_policy = DefaultBackdownPolicy;
 
+  /**
+   * @brief The strategy used for retry attempts.
+   *
+   * The backdown strategy used to compute new wait time to retry failed
+   * requests. By default it is not specified and, if specified, it will used
+   * instead \ref backdown_policy.
+   *
+   * @note You can use the \ref ExponentialBackdownStrategy from OLP SDK as new
```

Comment:
```suggestion
   * @note You can use the \ref ExponentialBackdownStrategy as the new
```
Redundant reference to the SDK